app/src/main/java/com/sdp/movemeet/ui/workout/SectionsPagerAdapter.java

```diff
@@ -0,0 +1,46 @@
+package com.sdp.movemeet.ui.workout;
+
+import android.content.Context;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.StringRes;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentPagerAdapter;
+
+import com.sdp.movemeet.R;
```

Comment:
What is the section pager adapter ?

Reply:
It organizes the workouts into 'tabs' which you can see on the top of the workout screen.